docs/pyapi/material_library.rst

```diff
@@ -0,0 +1,72 @@
+.. _pyne_material_library:
+
```

Comment:
Can you add info here, like you did in the docs for `material` that describes the file layout?  I've completely forgotten what we decided and need to be reminded to confirm this code.  It's also a good idea to document it.